content/en/docs/reference/kubectl/_index.md

```diff
@@ -538,16 +538,9 @@ the following contents:
 kubectl config view --template='{{ range .contexts }}{{ if eq .name "'$(kubectl config current-context)'" }}Current user: {{ printf "%s\n" .context.user }}{{ end }}{{ end }}'
 ```
 
-Running the above command gives you an output containing the user for the
-current context in your KUBECONFIG file:
+Make the plugin executable and move it to your PATH to use it successfully.
 
 ```shell
-# make the file executable
-sudo chmod +x ./kubectl-whoami
-
-# and move it into your PATH
-sudo mv ./kubectl-whoami /usr/local/bin
-
```

Comment:
Instead of fixing individual sentences, we can rearrange the order in which the information is presented to the user. For instance the line that says "The next few examples assume that you already made `kubectl-whoami` have
the following contents:" should come at the beginning, before we make the user run `cat ./kubectl-whoami`.

Reply:
I agree, I think the structure of this section can be revised to make it more clear.

Reply:
I'm open to making more changes here. Even after updating, I agree, this still looks a bit unclean-ish.